sdk/core/core-aborter/api-extractor.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "types/src/aborter.d.ts",
+  "docModel": {
+    "enabled": false
+  },
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "./review"
+  },
+  "dtsRollup": {
+    "enabled": true,
+    "publicTrimmedFilePath": "./types/aborter.d.ts"
```

Comment:
please add a line to disable generation of untrimmed dts rollup. otherwise it is generated by default into `dist/` and thus gets packaged.

```js
   "untrimmedFilePath": "",
```

See similar changes in https://github.com/Azure/azure-sdk-for-js/pull/3461/files